BrainPortal/app/controllers/nh_projects_controller.rb

```diff
@@ -52,6 +52,25 @@ def create #:nodoc:
     end
   end
 
+  def destroy #:nodoc:
+    @nh_project = current_user.modifiable_groups.find(params[:id])
+
+    if ! current_user.has_role?(:admin_user)
+      if current_user.id != @nh_project.creator_id
+        cb_error "Cannot destroy this project: you are not its creator.", :redirect => nh_project_path(@nh_project)
+      end
+    end
+    
+    @nh_project.destroy
```

Comment:
If the project cannot be destroyed, it would be nice if the user got the reasons, in the flash maybe.